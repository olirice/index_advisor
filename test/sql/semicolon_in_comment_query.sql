begin;
    -- Semicolons should be allowed in comments because they are common in prep stmts
    create extension index_advisor version '0.2.0' cascade;

    create table public.book(
        id int,
        name text
    );

    select index_advisor($$
        -- some comment with a semicolon;
        select * from book where id = $1
    $$);

rollback;
