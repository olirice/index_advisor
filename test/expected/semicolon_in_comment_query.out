begin;
    -- Semicolons should be allowed in comments because they are common in prep stmts
    create extension index_advisor version '0.1.2' cascade;
NOTICE:  installing required extension "hypopg"
    create table public.book(
        id int,
        name text
    );
    select index_advisor($$
        -- some comment with a semicolon;
        select * from book where id = $1
    $$);
                                index_advisor                                 
------------------------------------------------------------------------------
 (0.00,4.07,25.88,13.54,"{""CREATE INDEX ON public.book USING btree (id)""}")
(1 row)

rollback;
